extra-wm/i3status-rust/autobuild/build

```diff
@@ -0,0 +1,7 @@
+cargo build --release --locked
+install -Dm755 -t "${PKGDIR}"/usr/bin target/release/i3status-rs
```

Comment:
Please use verbosity for all commands, as required by the [AOSC OS Package Styling Manual](https://wiki.aosc.io/developer/packaging/package-styling-manual/#scripting).

Reply:
What does "verbosity" mean? Can I use `abinfo` ?